src/test/java/com/wikia/webdriver/elements/mercury/pages/CategoryPage.java

```diff
@@ -0,0 +1,146 @@
+package com.wikia.webdriver.elements.mercury.pages;
+
+import com.wikia.webdriver.common.contentpatterns.MercurySubpages;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.elements.common.Navigate;
+import com.wikia.webdriver.elements.mercury.components.Loading;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class CategoryPage extends WikiBasePageObject {
+
+  private By article = By.cssSelector(".article-content");
+  private By categorySections = By.cssSelector(".category-sections");
+  private By noMembersMessage = By.cssSelector(".category-page-no-members");
+  private By loadMoreButton = By.cssSelector("category-navigation__button[data-next]");
+  private By loadPreviousButton = By.cssSelector("category-navigation__button[data-previous]");
+
+  private Navigate navigate;
+
+  public CategoryPage() {
+    super();
+
+    navigate = new Navigate(driver);
+  }
+
+  public CategoryPage navigateToPageWithArticleAndWithMembersFromUrl() {
+    navigate.toPage(MercurySubpages.CATEGORY_WITH_ARTICLE_AND_WITH_MEMBERS);
+
+    articleContainerIsVisible();
+    categorySectionsContainerIsVisible();
+
+    return this;
+  }
+
+  public CategoryPage navigateToPageWithArticleAndWithoutMembersFromUrl() {
+    navigate.toPage(MercurySubpages.CATEGORY_WITH_ARTICLE_AND_WITHOUT_MEMBERS);
+
+    articleContainerIsVisible();
+    noMembersMessageIsVisible();
+
+    return this;
+  }
+
+  public CategoryPage navigateToPageWithoutArticleAndWithMembersFromUrl() {
+    navigate.toPage(MercurySubpages.CATEGORY_WITHOUT_ARTICLE_AND_WITH_MEMBERS);
+
+    articleContainerIsNotPresent();
+    categorySectionsContainerIsVisible();
+
+    return this;
+  }
+
+  public CategoryPage loadMoreMembersForSection(String name) {
+    navigateThroughSection(name, loadMoreButton);
+
+    return this;
+  }
+
+  public CategoryPage loadPreviousMembersForSection(String name) {
+    navigateThroughSection(name, loadPreviousButton);
+
+    return this;
+  }
+
+  public CategoryPage navigateToCategoryMemberPage(String name) {
+    WebElement member = driver.findElement(By.xpath("//a[contains(text(), \"" + name + "\")]"));
+    wait.forElementClickable(member);
+    member.click();
+
+    System.out.println(member);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS106)